common/src/main/resources/Operator.properties

```diff
@@ -178,6 +178,8 @@ WLSKO-0226=Pod {0} was evicted due to {1}; validating domain
 WLSKO-0227=Pod {0} was evicted due to {1} but the operator is configured not to restart it.
 WLSKO-0228=Watch event triggered for WebLogic Cluster {0} in WebLogic Domain with UID: {1}.
 WLSKO-0229=Watch event triggered for deletion of WebLogic Cluster {0} in WebLogic Domain with UID: {1}.
+WLSKO-0230=Null Cluster name.
```

Comment:
I'm going to be curious about how this is used... I would expect clusters to have a name or to default to the `metadata.name` of the Cluster resource. Similarly, we can make fields required in the schema.